internal/vmpool/google/config.go

```diff
@@ -0,0 +1,258 @@
+package google
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/drone-runners/drone-runner-aws/internal/cloudinit"
+
+	"github.com/drone-runners/drone-runner-aws/oshelp"
+
+	"github.com/buildkite/yaml"
+	"github.com/drone/runner-go/logger"
+
+	"github.com/drone-runners/drone-runner-aws/internal/vmpool"
```

Comment:
move this line up to the correct section.

Reply:
done